### ISSUE TITLE
OSX GUI off Screen

### ISSUE THREAD

**Issue by DjLeeAtlas** [bupticybee/TexasSolver#93]
Hi 

The GUI is bigger than my Macbook pro screen and I can't reduce size of it ..Is this a known issue ?

Thanks In advance

**Comment by bupticybee** [comment]
Yes, it's an known issue. I set a minimal window size to ensure all the  information can be displayed correctly.
